packages/anasazi/tpetra/test/LOBPCG/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 
-
 ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Triutils)
-IF (${PACKAGE_NAME}_ENABLE_Triutils)
+IF (${PACKAGE_NAME}_ENABLE_Triutils AND Tpetra_INST_COMPLEX_DOUBLE)
```

Comment:
Woah, would this have been a build error before if `Tpetra_INST_COMPLEX_DOUBLE` were OFF?  I'm OK with the increased specificity but just wondering why it seemed to work before.

Reply:
The ifdef based on Tpetra_INST_COMPLEX_DOUBLE was at the packages/anasazi/tpetra/test level and would ignore whole directories based on that ifdef.  That seemed too heavy-handed, so I moved the logic down into the individual solver test directories.